modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/auto/AutoConverter.kt

```diff
@@ -143,6 +149,23 @@ object AutoConverter {
         return getArtworkUriForContentProvider(podcastArtUri, context)
     }
 
+    fun getPodcastArtworkBitmap(episode: BaseEpisode, context: Context, useEpisodeArtwork: Boolean): Bitmap? {
+        val imageRequestFactory = PocketCastsImageRequestFactory(
+            context,
+            isDarkTheme = true,
+            size = 480,
+            placeholderType = PocketCastsImageRequestFactory.PlaceholderType.Small,
+        )
```

Comment:
I wasn't sure what size to send over. It can be displayed larger, but I didn't want to send too much data, so I picked the middle image size.

Reply:
Can you compare with a few other podcast apps? I noticed that Pocket Casts uses an image that's quite low quality in many places compared to other apps.